<launch>
  <!-- <arg name="cam_name" default="fsp_l" description="Options: fsp_l, lspf_r, lspr_l, rsp_l, rspf_l, rspr_r"/> -->

  <group>
    <push-ros-namespace namespace="sensor/camera"/>
    <node_container pkg="rclcpp_components" exec="component_container_mt" name="camera_inspector_container" namespace="">
      <composable_node pkg="av_camera_inspector" plugin="camera::TimestampInspector" name="cam_timestamp_inspector" namespace=""> 
        <!-- <extra_arg name="use_intra_process_comms" value="true"/> -->
      </composable_node>
    </node_container>
  </group>
</launch>
